narratives/ncov_sit-rep_2020-04-10.md

```diff
@@ -0,0 +1,6369 @@
+Tips of the tree represent samples from infected individuals.
+Each internal node, or branch point, represents an unsampled case, which was the ancestor of all of the other cases within the descendant clade.
+<br><br>
+Tips are colored according to where they were sampled from; nodes (branch points) are colored by the inferred (most likely) location of the ancestral case.
+<br><br>
+Comparing where a sequence was isolated from geographically to where its ancestors were circulating can help us understand not only where infections are occurring, but also where they may have been acquired.
+<br><br>
+The common ancestor of circulating viruses appears to have emerged in Wuhan, China, in late Nov or early Dec 2019.
+Accordingly, the majority of sampled and inferred ancestral cases were located in Asia during this early period.
+This is consistent with circulation largely within China and some exported cases to other countries in Asia for the first month of the outbreak.  
+
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Initial introductions to other regions](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-02-15&dmin=2020-01-15&p=full&d=map,tree)
+
+During the 2nd month of the outbreak, from mid-January to mid_February 2020, we start to see individual cases crop up in North America, Europe and Oceania.
+These early samples still group with samples from Asia, and generally lack the hallmarks of local transmission.
+<br><br>
+This is consistent with initial sparks being thrown from the main outbreak in Asia to other parts of the world. Although most of these sparks did not result in localized outbreaks, at least two of these introductions took off.
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Cryptic transmission in Europe from late January](https://nextstrain.org/ncov/global/2020-04-09?d=tree,map&dmax=2020-02-14&label=clade:A2&p=full)
+
+Zooming in to the top of the tree for the same time period, we see an ancestral European virus (internal node) that seeded the large European outbreak.
+<br><br>
+This indicates that while we hadn't yet sequenced any samples from this clade, the origins of the European outbreak were already circulating undetected by late January.
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Cryptic transmission in North America from late January](https://nextstrain.org/ncov/global/2020-04-09?d=tree,map&dmax=2020-02-14&label=clade:B1&p=full)
+
+Similarly, zooming in to the bottom of the tree, we see an ancestral North American virus (internal node) that initiated what would later become a large outbreak in North America.
+<br><br>
+This indicates that although we only had 1 sequenced case from this cluster at the time, the virus was circulating undetected within North America by late January.
```

Comment:
Again, I'd say 'late Jan to early Feb'. If we look at the North American build (with more sequences) the confidence intervals are wider on this clade and overlap with early Feb. That's worth taking into account, I think! And I'd rather us be more unsure in the outset then have to walk back later!